src/routes/+page.svelte

```
@@ -1,89 +1,53 @@
-	import ArticleSpotlight from '$lib/components/ui/ArticleSpotlight.svelte';
-	import ArticleList from '$lib/components/ui/ArticleList.svelte';
-	import type { PageData } from './$types';
-	import { setArticles } from '$lib/stores/articles.svelte';
-	import { onMount } from 'svelte';
-	import Testimonials from '$lib/components/ui/Testimonials.svelte';
-	import BaseHead from '$lib/components/server/BaseHead.svelte';
-	import Button from '$lib/components/ui/button/button.svelte';
-	import { ArrowRight, Mail } from 'lucide-svelte';
-	import ArticleCategory from '$lib/components/ui/ArticleCategory.svelte';
+    import ArticleSpotlight from '$lib/components/ui/ArticleSpotlight.svelte';
+    import ArticleList from '$lib/components/ui/ArticleList.svelte';
```

ESLint

[error] 'ArticleList' is defined but never used.

(@typescript-eslint/no-unused-vars)